### PR TITLE
Network ios

### PR DESCRIPTION
**Pull request by viikufa**

### CODE REVIEW

File: station.localization.json [hunk 1]
```diff
@@ -3841,7 +3850,7 @@
       "ident_android": "sign_in_benefits_description",
       "en": "After signing in, you will be automatically marked as an owner of your nearby Ruuvi sensors. The claimed sensors will be restored after an app re-install.\\n\\nWith a Ruuvi Gateway router, you'll be able to monitor the claimed sensors remotely and share them with your friends.",
       "fi": "",
-      "ru": "",
+      "ru": "Войдя в сеть вы сможете сохранить свои данные в облаке:\\n- список датчиков\\n- названия датчиков",
```
Comment (rinat-enikeev) [inline station.localization.json:3853]: comma

File: station.localization.json [hunk 2]
```diff
@@ -3850,7 +3859,7 @@
       "ident_android": "sign_in_benefits_title",
       "en": "Benefits of Ruuvi Network",
       "fi": "",
-      "ru": "",
+      "ru": "Зарегистрируйтесь или получите ссылку для входа",
```
Comment (rinat-enikeev) [inline station.localization.json:3862]: are you sure?

File: station.localization.json [hunk 3]
```diff
@@ -4115,5 +4162,233 @@
...
+      "en": "verification code in format: CJSM",
+      "fi": "",
+      "ru": "код подтверждения в формате: CJSM",
+      "sv": "",
+      "fr": ""
+    },
+    {
+      "ident_ios": "UserApiError.ER_FORBIDDEN",
+      "ident_android": "",
+      "en": "Forbiden",
```
Comment (rinat-enikeev) [inline station.localization.json:4207]: Forbidden

File: station.localization.json [hunk 4]
```diff
@@ -4115,5 +4162,233 @@
...
+      "en": "Subscription is not found",
+      "fi": "",
+      "ru": "Подписка не найдена",
+      "sv": "",
+      "fr": ""
+    },
+    {
+      "ident_ios": "UserApiError.ER_SHARE_COUNT_REACHED",
+      "ident_android": "",
+      "en": "Share count is reached",
```
Comment (rinat-enikeev) [inline station.localization.json:4279]: share limit?

File: station.localization.json [hunk 5]
```diff
@@ -4115,5 +4162,233 @@
...
+      "fi": "",
+      "ru": "Достигнут лимит количества общих сенсоров",
+      "sv": "",
+      "fr": ""
+    },
+    {
+      "ident_ios": "UserApiError.ER_SENSOR_SHARE_COUNT_REACHED",
+      "ident_android": "",
+      "en": "Sensor share count is reached",
```
Comment (rinat-enikeev) [inline station.localization.json:4288]: limit?

File: station.localization.json [hunk 6]
```diff
@@ -4115,5 +4162,233 @@
...
+      "ru": "Превышено количество общих сенсоров",
+      "sv": "",
+      "fr": ""
+    },
+    {
+      "ident_ios": "UserApiError.ER_NO_DATA_TO_SHARE",
+      "ident_android": "",
+      "en": "No data to share sensor",
+      "fi": "",
+      "ru": "Нет данных для того чтобы поделиться сенсором",
```
Comment (rinat-enikeev) [inline station.localization.json:4299]: comma

File: station.localization.json [hunk 7]
```diff
@@ -4115,5 +4162,233 @@
...
+      "sv": "",
+      "fr": ""
+    },
+    {
+      "ident_ios": "UserApiError.ER_SENSOR_ALREADY_SHARED",
+      "ident_android": "",
+      "en": "This sensor already is shared",
```
Comment (rinat-enikeev) [inline station.localization.json:4306]: is already shared

File: station.localization.json [hunk 8]
```diff
@@ -4115,5 +4162,233 @@
...
+      "en": "Invalid email address",
+      "fi": "",
+      "ru": "Недопустимый e-mail",
+      "sv": "",
+      "fr": ""
+    },
+    {
+      "ident_ios": "UserApiError.ER_INVALID_MAC_ADDRESS",
+      "ident_android": "",
+      "en": "Invalid mac address",
```
Comment (rinat-enikeev) [inline station.localization.json:4369]: MAC

**Review by rinat-enikeev — CHANGES_REQUESTED** [review]

File: station.localization.json (hunk 1 above)
Comment (andreevdenis) [reply station.localization.json:3853]: English text was changed significantly here:

photo_2021-02-23_10-24-46

File: station.localization.json (hunk 2 above)
Comment (viikufa) [reply station.localization.json:3862]: nope, incorrect resolved merge conflict

File: station.localization.json [hunk 9]
```diff
@@ -3881,12 +3890,39 @@
...
+      "sv": "",
+      "fr": ""
+    },
     {
       "ident_ios": "SignIn.CheckMailbox",
       "ident_android": "sign_in_check_email",
       "en": "Check your email for an activation code.",
       "fi": "",
-      "ru": "Пожалуйста проверьте email для получения кода активации",
+      "ru": "Пожалуйста проверьте email для дальнейших инструкций",
```
Comment (rinat-enikeev) [inline station.localization.json:3925]: comma

File: station.localization.json [hunk 10]
```diff
@@ -3841,7 +3850,7 @@
       "ident_android": "sign_in_benefits_description",
       "en": "After signing in, you will be automatically marked as an owner of your nearby Ruuvi sensors. The claimed sensors will be restored after an app re-install.\\n\\nWith a Ruuvi Gateway router, you'll be able to monitor the claimed sensors remotely and share them with your friends.",
       "fi": "",
-      "ru": "",
+      "ru": "После входа в систему вы будете автоматически отмечены как владелец ближайших датчиков Ruuvi. Заявленные датчики будут восстановлены после повторной установки приложения. \\n\\nС помощью Ruuvi Gateway вы сможете удаленно контролировать заявленные датчики и делиться ими со своими друзьями.",
```
Comment (rinat-enikeev) [inline station.localization.json:3853]: we are using "сенсор" everywhere, let's use it here too @andreevdenis ?

**Review by rinat-enikeev — CHANGES_REQUESTED** [review]

File: station.localization.json (hunk 9 above)
Comment (rinat-enikeev) [reply station.localization.json:3925]: https://russkiiyazyk.ru/punktuatsiya/pozhaluista-vyideliaetsia-zapiatymi.html

File: station.localization.json (hunk 10 above)
Comment (andreevdenis) [reply station.localization.json:3853]: I agree

**Review by rinat-enikeev — APPROVED** [review]

**Pull request merged**